php7.2-fpm-nginx/nginx.vh.default.conf

```diff
@@ -10,7 +10,7 @@ server {
 	}
 
 	location ~ ^/index\.php(/|$) {
-		fastcgi_pass unix:/var/run/php7.2-fpm.sock;
+        fastcgi_pass 127.0.0.1:9000;
```

Comment:
offset nesedi

Reply:
to iba nejak nachuja zobrazuje ked to tam editnut tak je to ok